dominion/templates/profile/profile_detail.html

```diff
@@ -56,6 +24,22 @@
       <script src="{% static 'js/profile_map.js' %}"></script>
       <script src="{% static 'js/maps_mapit.js' %}"></script>
       <script src="{% static 'js/profile_map_mapit.js' %}"></script>
+      <script src="{% static 'js/map_mapit.js' %}"></script>
+
+          <script>
+            var maps = new ProfileMaps();
+            var zoom = {% if HURUMAP.map_zoom %}{{ HURUMAP.map_zoom }}{% else %}null{% endif %};
+
+            var COUNTRY = {{ geography.this|jsonify|safe }};
+            var centre = {
+                'country-KE': [-0.0236, 37.9062],
+                'country-ZA': [-30.5595, 22.9375],
+                'country-TZ': [-6.3690, 34.8888],
+                'country-NG': [9.0820, 8.6753],
+            }
+
+            maps.drawMapForHomepage('{{ geo_data.first_child_level }}', '{{ HURUMAP.default_geo_version }}', centre['{{ geography.this.full_geoid }}'], zoom);
```

Comment:
`profile_detail ` from hurumap also defines `a ProfileMaps` object. I think we should move to it's own template, once we compile a complete country page